app/src/main/java/com/github/fribourgsdp/radio/User.kt

```diff
@@ -112,6 +111,15 @@ data class User (var name: String, val color: Int) {
         playlists.remove(playlist)
     }
 
+    /**
+     * removes a single playlist given by its name from the user's set of playlists
+     *
+     * @param name the name of the playlist to remove
+     */
+    fun removePlaylistByName(name: String) : Boolean{
+        return playlists.removeIf{p -> p.name == name}
+    }
```

Comment:
If several playlists have the same name, they will all be deleted? If so, this is not consistent with the documentation of the function.

Reply:
Oh yes you're right, I shall update the documentation. In fact, name is the only thing that defines a playlist locally, its "primary key" to speak in term of db. So yes it might delete multiple ones if they have same name

Reply:
Should not happen because I think playlists are in a set at least before being passed to recyclerviews